workbench-core/example/app/src/configs/staticRouteConfig.ts

```diff
@@ -107,6 +107,15 @@ export const routesMap: RoutesMap = {
       }
     ]
   },
+  '/roles/\\S+': {
```

Comment:
Will this match special characters as well ?

Reply:
It matches all non-whitespace characters. I'm not 100% happy with how this is implemented, so if you have a suggestion on how to better organize the route I'd love to hear it!

Reply:
is this for matching roleNames ? and roleNames will be just alphabets then we can at least restrict it to match alphabets only [A-Za-z]+ ?

Reply:
These are the requirements for Cognito groups:
```
    The name of the group. Must be unique.

    Type: String

    Length Constraints: Minimum length of 1. Maximum length of 128.

    Pattern: [\p{L}\p{M}\p{S}\p{N}\p{P}]+

    Required: Yes
```
The regex I believe is equivalent to what I have, just not the length

Reply:
\S{1,128} ?

Reply:
I'll change in a followup PR